src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const Tabs = ({ tabs, onTabSelected, selectedTabId }) => (
+  <ul>
+    {tabs.map(
+      tab => (
```

Comment:
```suggestion
    {tabs.map(tab => (
```
Не вижу повода это переносить